resources/popup.js

```diff
@@ -1,27 +1,29 @@
+// Use the toggle button on the popup to switch the loggingEnabled option.
+function toggle() {
```

Comment:
Yeah, you have a point there, as in options.js you'd get a function of a few hundred lines long for the callback of `$(document).ready`.. :sweat_smile: Okay, you can leave it this way :)